### ISSUE TITLE
Scrollbar blinking in QScrollArea in drawer after scrolling down to the end

### ISSUE THREAD

**Issue by gkraser** [quasarframework/quasar#11169]
Scrollbar blinking in QScrollArea in drawer after scrolling down to the end.

Reproduced in Chrome (last version). In Firefox - all ok.

**To Reproduce**
1. Go to https://quasar.dev/layout-builder
2. In right drawer: scroll down to the end.
3. See problem

**Screenshots**
<details>
  <summary>Animated gif screenshot</summary>

![JE56SdaZGg](https://user-images.githubusercontent.com/1287745/139572373-dadf081f-c4fa-4d2b-bee1-3bf959526090.gif)

</details>

**Platform**
Quasar Version: 2.2.2
Windows 10
Chrome 95.0.4638.69

**Comment by MilosPaunovic** [comment]
Check out this discussion with the same problem [https://github.com/quasarframework/quasar/discussions/11062](https://github.com/quasarframework/quasar/discussions/11062)

**Comment by gkraser** [comment]
> Check out this discussion with the same problem #11062

Unfortunately, this did not solve my problem

**Comment by pdanpdan** [comment]
Are you using zoom on windows? (HiDPI or how it is called?)

**Comment by gkraser** [comment]
I have reproduced this problem in my application. This is how it looks like:

<details>
  <summary>animated gif</summary>

![PGJpFSx68a](https://user-images.githubusercontent.com/1287745/139583767-0236f06b-a861-465d-9b43-7fe352dfb0ed.gif)

</details>

**Comment by gkraser** [comment]
> Are you using zoom on windows? (HiDPI or how it is called?)

Windows 10, screen resolution: 1920x1080, scale 125%

**Comment by gkraser** [comment]
The header uses components that have dimensions in em/rem. As a result, fractional numbers in sizes px are obtained.

in chrome:

* body: 471.200
* header: 64.237
* page container: 405
* $q.screen.height: 471

in firefox:
 
* body: 471.2
* header: 64.25
* page container: 406
* $q.screen.height: 471

page container height different in chrome and firefox

**Comment by robfordww** [comment]
@gkraser  I can confirm the problem on my newly installed win11 laptop with chrome.

**Comment by vneznaikin** [comment]
is there a way to override the layout calculation? I found the only solution to change the values through "nextTick". but this is a not good solution. more like a temporary crutch

my temporary solution:
```
mounted() {
    this.$nextTick(() => {
      this.$refs.qLayout.$el.style.minHeight = "100vh"
    })
  },
```
do not forget add qLayout ref
and you also need to add resize listner

UPDATE:
nextTick it's bad solution
```
<q-layout view="hHh lpR fFf" style="min-height:100vh">
...
  <router-view style="margin-top:-1px"/>
```
little bit better but still not so good

**Comment by birkanucan** [comment]
> I have reproduced this problem in my application. This is how it looks like:
> 
> animated gif

I also observe this behaviour on quasar.dev (sections that have q-drawer-container on the right side e.g. https://quasar.dev/vue-components/ajax-bar)
I'm on win 10, 4k, scale: 150%. I see it on chrome and edge when window is maximized and on certain points when I drag and change window size from the corner, doesn't seem to happen on firefox dev edition at all. 
It stops blinking when I remove scroll from "q-drawer__content fit scroll" on devtools and removal doesn't seem to break it.

**Comment by hawkeye64** [comment]
> > I have reproduced this problem in my application. This is how it looks like:
> > animated gif
> 
> I also observe this behaviour on quasar.dev (sections that have q-drawer-container on the right side e.g. [quasar.dev/vue-components/ajax-bar](https://quasar.dev/vue-components/ajax-bar)) I'm on win 10, 4k, scale: 150%. I see it on chrome and edge when window is maximized and on certain points when I drag and change window size from the corner, doesn't seem to happen on firefox dev edition at all. It stops blinking when I remove scroll from "q-drawer__content fit scroll" on devtools and removal doesn't seem to break it.

I think you can just remove `scroll`. Does your drawer also contain a QScrollArea?

**Comment by birkanucan** [comment]
I'm not working on an application, the comment I've quoted is not mine. 😅 I was just checking the docs (quasar.dev) and came upon it. But yes, it seems it does. https://i.imgur.com/FiINF4L.gif

**Comment by hawkeye64** [comment]
What your zoom level for your monitor?
@birkanucan

**Comment by hawkeye64** [comment]
Nvm, I see it above

**Comment by rstoenescu** [comment]
Fixed it in Quasar v2.4.4 & v1.17.1 (to be released)

**Comment by gkraser** [comment]
> Fixed it in Quasar v2.4.4 & v1.17.1 (to be released)

Thank you, it works correctly!
